drulenium_visual_testing/post_drulenium_github.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+// An example of usign Pantheon's Quicksilver technology to do 
+// automatic visual regression testing using Drulenium
+
+// Modify default Test definition accordingly.
+/* Example
+$test_definition = array (
+  'staging_environment_domain' => 'http://test-drulenium-hosted.pantheonsite.io/',
+  'dev_environment_domain_suffix' => '-drulenium-hosted.pantheonsite.io/',
+  'notification_email_ids' => array(
+    'drulenium@mailinator.com',
+  ),
+  'test_pages_sitemap' => array(
+    'boston',
+    'new-york',
+    'node',
+  ),
+);
+*/
+$test_definition = array (
+  'staging_environment_domain' => 'http://test-<add-my-site-name>.pantheonsite.io/',
+  'dev_environment_domain_suffix' => '-<add-my-site-name>.pantheonsite.io/',
```

Comment:
@ndubbaka Have you considered replacing `<add-my-site-name>` with `$_ENV['PANTHEON_SITE_NAME']`? See our [doc on environment configuration](https://pantheon.io/docs/read-environment-config/) for available values